op-devstack/presets/interop.go

```diff
@@ -116,3 +116,26 @@ func WithInteropNotAtGenesis() stack.CommonOption {
 		}
 	})
 }
+
+type RedundancyInterop struct {
+	SimpleInterop
+
+	L2ELA2 *dsl.L2ELNode
+	L2CLA2 *dsl.L2CLNode
+}
+
+func WithRedundancyInterop() stack.CommonOption {
+	return stack.MakeCommon(sysgo.DefaultRedundancyInteropSystem(&sysgo.DefaultRedundancyInteropSystemIDs{}))
+}
+
+func NewRedundancyInterop(t devtest.T) *RedundancyInterop {
+	simpleInterop := NewSimpleInterop(t)
```

Comment:
If we're using simpleInterop to create this system why not add the replica to the the SimpleInterop struct?

Reply:
SimpleInterop (the struct) is just a frontend. We can match the same set of nodes as the smaller frontend, and match additional ones only here. It'll skip (with `match.Assume`) or error (regular match) if there is no matching resource found. We still want the regular smaller tests to be able to run against the small system, so we can't add the matching to the small backend. I think the composition approach by CW is pretty good.